test/word_list_test.rb

```diff
@@ -6,4 +6,14 @@
 
 class WordListTest < Minitest::Test
 
+
+  def test_map_letters_to_one
+    wordlist = WordList.new
+
+
+
+    assert_equal {"A", "E": 1}
+
+
+  end
```

Comment:
No need to touch this file. All the functionality described in the interaction pattern revolves around the Scrabble class.